tests/playwright/ui-tests/cookies.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test } from '@playwright/test'
+import { HomePage } from '../page-object-model/home-page'
+import { CookiesPage } from '../page-object-model/cookies-page'
+import { CurrentSearch } from '../page-object-model/shared/search-form-component'
+
+test.describe('cookiespage', () => {
+  let homePage: HomePage
+  let cookiesPage: CookiesPage
+  let currentSearch: CurrentSearch
+
+  test.beforeEach(async ({ page, context }) => {
+    currentSearch = new CurrentSearch()
+    homePage = new HomePage(page, currentSearch)
+    cookiesPage = new CookiesPage(page)
+    await homePage.goTo()
+  })
+
+  test('user shoud be able to accept cookies at cookies page', async () => {
+    await homePage.footerNavigation.clickCookies()
+    await cookiesPage.expect.returnToLinkPageToNotBeVisble()
+    await cookiesPage.acceptCookies()
+    await cookiesPage.expect.returnToLinkPageToBeVisble()
+    await cookiesPage.clickBackToHomePage()
+    await homePage.expect.toBeOnTheRightPage()
+    await homePage.footerNavigation.clickCookies()
+    await cookiesPage.expect.acceptCookiesRadioButtonIsChecked()
+  })
```

Comment:
We should also have an assertion that the cookies are actually removed. In the page object model we can get access to the actual cookies through `await page.context().cookies()`